### DISCUSSION TITLE
Invitation to users for usability testing

### DISCUSSION THREAD

**Discussion by DianaAvilesCh** [ransome1/sleek#396] [General]
Dear Users,

Our names are Diana Aviles, Andrés Mora and Lister Mera. We have started a course project with the title: "Adoption of the Help Organization for Use Case Technique in the Open-Source Software Development Process" at the State Technical University of Quevedo.

When working with Open Source Software, you have probably noticed that some applications are not easy, fast and accessible to use. With this research, empirical evidence was provided to solve many aspects of usability unknown to the Sleek community. To achieve this goal, first of all your collaboration is required, responding to a survey with the purpose of finding out what the characteristics of the possible Sleek user profiles are.

At the end of the survey, the responses will be analyzed. The result of this analysis will be shared with the Sleek developer community to improve the user experience with Sleek.

The survey is anonymous and will take up to 15 minutes to complete. Here is the link to the survey:

[Form](https://forms.gle/9wvYLvQaFAVutPFQ6)

The deadline to complete the survey is July 22, 2022. We hope, please, answer the survey as soon as possible to continue our investigation. We greatly appreciate the time you will take to fill out the survey. Many thanks.

Best regards,

Diana Aviles
Andres Mora
Lister Mera

**Comment by Saturo78** [comment]
Required authorization is missing

Are you sure that this isn´t spam?
